scubagoggles/auth.py

```diff
@@ -14,7 +14,6 @@
 # If modifying these scopes, delete the file token.json.
 SCOPES = ['https://www.googleapis.com/auth/admin.reports.audit.readonly',
     "https://www.googleapis.com/auth/admin.directory.domain.readonly",
-    "https://www.googleapis.com/auth/admin.directory.customer.readonly",
```

Comment:
Can you also remove this from the readme?

Reply:
Good catch